docs/develop/security.md

```diff
@@ -110,32 +110,98 @@ contract Secret {
 
 ## Gas Padding
 
-To prevent leaking information about a particular transaction, Sapphire
-provides a [precompile] for dApp developers to **pad the amount of gas used
-in a transaction**.
+Gas padding lets you equalize **EVM execution** gas across private code paths to reduce side‑channel leakage. Sapphire provides a [precompile] (`Sapphire.padGas`) that burns execution gas so that your function’s execution cost is brought up to a target amount.
```

Comment:
You can add `Sapphire.padGas` link to https://api.docs.oasis.io/sol/sapphire-contracts/contracts/Sapphire.sol/library.Sapphire.html#padgas

Reply:
Maybe add another sentence

```markdown
The gas padding call is usually done somewhere at the end of the executed code to cover all possible execution paths.
```

Because the function doesn't "hardcode" the total gas spent, but just burns it to the specific amount and then continues with the execution and spends gas normally on.